test/util.py

```diff
@@ -60,26 +61,33 @@ def f_impl(*args, **kwargs):
 
         return f
 
+
     def runBoth(self, func, *inputs, check_tvm=True):
-        # jit the function
-        trace_jit = torch.jit.trace(func, inputs)
-        ref_out = trace_jit(*inputs)
-
-        # jit the function and lower to TVM
-        torch_tvm.enable()
-        trace_tvm = torch.jit.trace(func, inputs)
-        tvm_out = trace_tvm(*inputs)
-
-        if check_tvm == True:
-            tvm_unused = "TVM was not able to optimize this trace."
-            assert "tvm::CompilationGroup" in str(
-                trace_tvm.graph_for(*inputs)
-            ), tvm_unused
-            # tvm compile the graph and ensure TVM is used
-            with profile() as p:
-                _ = trace_tvm(*inputs)
-            assert "TVM" in [_.name for _ in p.function_events], tvm_unused
-
-        torch_tvm.disable()
-
-        return ref_out, tvm_out
+        with torch.no_grad():
+          # jit the function
+          trace_jit = torch.jit.trace(func, inputs)
+          ref_out = trace_jit(*inputs)
+
+          # jit the function and lower to TVM
+          torch_tvm.enable()
+          with autotvm.apply_history_best("test/autotvm_tuning.log"):
```

Comment:
this actually will not be that transparent to the user. I think we might can make the `torch_tvm.enable` API take if we do autotvm and pass the log there, in the future we will need to think about how much tunable tvm api that we want to expose to the user.

Reply:
everything is exposed to the user by default pretty much